src/pages/CreateSalary/CreateSalary.tsx

```diff
@@ -1,120 +1,270 @@
-            className="input"
-            value={formNewSalary.position}
-          ></IonInput>
-        </IonItem>
-        <br />
-        <IonItem
-          class="ion-justify-content-center ion-padding-end"
-          lines="none"
-        >
-          <IonRange
-            class="ion-no-padding ion-padding-vertical "
-            pin={true}
-            onIonChange={(e) =>
-              setFormNewSalary({
-                ...formNewSalary,
-                salary: e.detail.value as number,
-              })
-            }
-            min={1000}
-            max={2000}
-            color="secondary"
+      <IonContent class="">
+        <IonGrid className="page-header" fixed>
+          <IonButton
+            class="ion-hide-sm-down"
+            fill="clear"
+            routerLink="/salaries"
           >
-            <IonLabel slot="start" class="ion-margin-end">
-              Salario
-            </IonLabel>
-            <IonLabel slot="start">1000€</IonLabel>
-            <IonLabel slot="end">3000€</IonLabel>
-          </IonRange>
-        </IonItem>
+            <IonIcon class="ion-margin-end" icon={arrowBack}></IonIcon> Back to
+            Salaries
+          </IonButton>
+          <IonList class="form-list ion-padding">
+            <div className="input-wrapper">
+              <IonLabel class="label">
+                Experience in the position (years)
+              </IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    experience: parseInt(e.detail.value!, 10),
+                  })
+                }
+                type="number"
+                className="input ion-margin-top"
+                value={formNewSalary.experience}
+              ></IonInput>
+            </div>
+            <div className="input-wrapper">
+              <IonLabel class="label">Current Position</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    position: e.detail.value! as string,
+                  })
+                }
+                type="text"
+                className="input ion-margin-top"
+                value={formNewSalary.position}
+              ></IonInput>
+            </div>
+            <div className="input-wrapper">
+              <IonLabel class="label">Role</IonLabel>
+              <IonInput
+                onIonChange={(e) =>
+                  setFormNewSalary({
+                    ...formNewSalary,
+                    role: e.detail.value! as string,
+                  })
+                }
+                className="input ion-margin-top"
+                value={formNewSalary.role}
+                type="text"
+              ></IonInput>
+            </div>
+            <div className="input-wrapper">
+              <IonLabel class="label">Employment Type</IonLabel>
```

Comment:
For the fields that will be a list of options, will we change them in a following PR?

Reply:
That is a good question, we though about iterating on the form but not sure if it will be on the on the same PR. This needs to be discussed.

Reply:
We can do it in another PR, I think this is a great starting point!